src/main/java/uk/gov/ch/oauth/IOAuthCoordinator.java

```diff
@@ -1,11 +1,25 @@
 package uk.gov.ch.oauth;
 
-import java.util.Map;
-import javax.servlet.http.HttpServletResponse;
 import uk.gov.ch.oauth.exceptions.UnauthorisedException;
+import uk.gov.companieshouse.session.Session;
+
+import javax.servlet.http.HttpServletResponse;
+import java.util.Map;
 
 public interface IOAuthCoordinator {
 
     String getPostCallbackRedirectURL(HttpServletResponse response, Map<String, String> params)
             throws UnauthorisedException;
+
+    /**
+     * @return The redirect Uri so that it can be assigned to a parameter for the sendRedirect() method
```

Comment:
May as well add a description. However I shouldn't have forgotten to do this for Callback. Could I ask you to cover that as well?

Reply:
Ive added that in now :)